data/hai/hai news.txt

```diff
@@ -0,0 +1,251 @@
+			"I saw the Hai-Human fusion band Nuclear Martini playing on Luna. Nobody realized the hairy drummer was really a Hai in disguise."
+			"I saw a Hai wearing the cutest bonnet the other day. When I get back to human space, I'm definitely getting a pet squirrel."
+			"Do you see that Hai woman carrying her baby? Doesn't it look like a long-tailed teddy-bear? The Hai are so cute."
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "concerned merchant in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Merchant captain"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I took a shortcut through Wah Ki with six ships. Just two seconds later, BAM! half my fleet is dust."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
+			"These Hai are so creepy. What are they hiding? I can't wait to get back to human space."
+			"The Hai don't like my ship's armory of laser rifles, but I was once boarded by a pirate crew and I'm never letting that happen again! If some overgrown squirrel is going to take away my rifle, it'll have to be from my cold dead hands!"
+			"I met some young Hai pilots. It sounds like they start out as poor as we do."
+			"I hope nobody is smuggling Hai technology out of here. I don't want to think about what would happen if the pirates got their hands on this stuff."
+			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
+
+# Visual suggestion: a bit quirky or weird; expression is content, confident, or stoic
+news "human citizen in Hai space"
+	location
+		government "Hai"
+	name
+		word
+			"Human citizen"
+	portrait
+		portrait/human02
+		portrait/human04
+		portrait/human06
+		portrait/human10
+		portrait/human15
+		portrait/human17
+		portrait/human31
+		portrait/human58
+		portrait/human63
+		portrait/human72
+		portrait/human73
+		portrait/human76
+		portrait/human79
+		portrait/human98
+		portrait/human108
+		portrait/human112
+		portrait/human125
+	message
+		word
+			"Don't go too far north up here. The Hai have their own pirates, and they're much more powerful than anything we're used to in human space."
+			"I hear Hai space wasn't peaceful a few centuries ago when some reptiles were raiding the place. I wonder where they went?"
+			"My Hai friend went on one of those prison cruises a while back, but it got lost in an ion storm near Allhome."
```

Comment:
What in the world is an ion storm? (We don't have system hazards yet.)

Reply:
This is to set up lore for the Philosopher's Paradise story. The excuse for the loss is intentionally unbelievable; the prison cruise was a trick and the vacationers were kidnapped.

Reply:
A fine point. I did all of this just to set up the lore for the Philosopher's Paradise story. Arachi was concerned that there weren't even rumors about people going missing. Adding such rumors belongs in that PR, not this one.

Reply:
I removed this news story. I'll keep it in the Philosopher's Paradise PR.